srcMain/main.cpp

```diff
@@ -228,16 +228,20 @@ void shasta::main::assemble(
 
 
 
-    // Create the run the output directory. If it exists, stop.
-    if(filesystem::exists(assemblerOptions.commandLineOnlyOptions.assemblyDirectory)) {
-        throw runtime_error(
-            "Assembly directory " +
-            assemblerOptions.commandLineOnlyOptions.assemblyDirectory +
-            " already exists.\n"
-            "Remove it or use --assemblyDirectory to specify a different assembly directory.");
+    // Create the run the output directory. If it exists and is not empty then stop.
+    bool dirExists = filesystem::exists(assemblerOptions.commandLineOnlyOptions.assemblyDirectory);
```

Comment:
Also check that it is a directory, with a call to `shasta::filesystem::isDirectory`?

Reply:
Looks like `shasta::filesystem::isDirectory` does the exists check as well. Great.